package.json

```diff
@@ -13,7 +13,14 @@
         "onLanguage:scss",
         "onLanguage:sass",
         "onLanguage:less",
-        "onLanguage:vue"
+        "onLanguage:vue",
+        "onLanguage:javascript",
+        "onLanguage:typescript",
+        "onLanguage:javascriptreact",
+        "onLanguage:typescriptreact"
+    ],
+    "extensionDependencies": [
+        "zardoy.ts-essential-plugins"
```

Comment:
Dependencies has a lot of known issues (eg pylance using it and no one likes it) so they should never be used.

Also don't see a need to force users install it, let's include a note to readme